pkgs/top-level/all-packages.nix

```diff
@@ -42119,4 +42119,6 @@ with pkgs;
   ssl-proxy = callPackage ../tools/networking/ssl-proxy { };
 
   code-maat = callPackage ../development/tools/code-maat {};
+
+  cowsql = callPackage ../development/libraries/cowsql {};
```

Comment:
consider initializing this in `pkgs/by-name` (same for incus, which seems missing from `all-packages`)